src/core/time.d

```diff
@@ -1667,6 +1667,52 @@ public:
         }
     }
 
+    /++
+        Converts `Duration` to a `timeval` structure.
+    +/
+    timeval toTimeVal() const nothrow @nogc
+    {
+        timeval value;
+        split!("seconds", "usecs")(value.tv_sec, value.tv_usec);
+        return value;
+    }
+
+    unittest
```

Comment:
Please add `///` on the line before, to make this into a documented example (https://dlang.org/spec/ddoc.html#using_ddoc_to_generate_examples).